### PR TITLE
Setup portfolio skeleton by making Navbar and Header Section

### PR DESCRIPTION
**Pull request by Metaverse-Nabeel**
Following tasks are done in this project.
1. Made the navbar and header section for the portfolio website using the Figma design
2. Checked for the linter error and corrected them.
3. Made README.md file with LICENSE file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<body>
+    <nav class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <div class="navbarItems">
+            <a href="#">Portoflio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+        </div>
+        <img class="hamburger" src="./Assets/images/bars.svg" alt="hamburger">
+    </nav>
```
Comment (Zuhaib042) [inline index.html:22]: - [ ] Kindly consider taking a `header` tag directly under `body` tag. that `header` tag would have two children : an `a` tag for logo and a `nav` tag that would consist of the hamburger menu. the reason for that is to follow best practice and conventions, better semantics, and better user experience.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Metaverse-Nabeel , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights
✅ Correct GitHub flow
✅ Professional Readme
✅ Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Metaverse-Nabeel) [reply index.html:22]: Hi @Zuhaib042,

I have made the changes asked as per your comment and requesting for the review again.

Thank you for valuable feedback

Regards

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+<body>
+    <header class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <nav>
+            <div class="navbarItems">
+            <a href="#">Portoflio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+            </div>
```
Comment (GedeonTS) [inline index.html:21]: - [OPTIONAL] Good job adding more than required for this milestone, I suggest using an unordered list to layout this menu in order to meet the semantic HTML rules

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @Metaverse-Nabeel ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of Github flow :+1: 


## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Metaverse-Nabeel) [reply index.html:21]: Hi @GedeonTS,

Thank you for your kind feedback. I shall make the optional changes recommended by you. I shall make the git merge now.

Regards

**Pull request merged**